e2e/cypress/integration/emoji/custom_emoji_spec.js

```diff
@@ -34,13 +80,520 @@ describe('Custom emojis', () => {
+            cy.findAllByTestId('emojiItem').children().should('have.length', 1);
+            cy.findAllByTestId('emojiItem').children('img').first().should('have.class', 'emoji-category--custom');
+
+            cy.findAllByTestId('emojiItem').children().click();
+
+            // * Complete the edit post action
+            cy.get('#editButton').click();
+        });
+
+        // # Verify that the builtin emoji is present in the post
+        cy.getLastPost().find('p').should('contain.html', messageText + ' <span data-emoticon="' + builtinEmojiWithoutColons + '"><span alt="' + builtinEmojiWithColons + '" class="emoticon" title="' + builtinEmojiWithColons + '" style="background-image: url(&quot;' + builtinEmojiImageUrl + '&quot;);">' + builtinEmojiWithColons + '</span>');
+
+        // # Verify that the custom emoji is present in the post
+        cy.getLastPost().find('p').should('contain.html', '<span data-emoticon="' + customEmojiWithoutColons + '">');
+
+        cy.reload();
+
+        // # Verify that the builtin emoji is present in the post
+        cy.getLastPost().find('p').should('contain.html', messageText + ' <span data-emoticon="' + builtinEmojiWithoutColons + '"><span alt="' + builtinEmojiWithColons + '" class="emoticon" title="' + builtinEmojiWithColons + '" style="background-image: url(&quot;' + builtinEmojiImageUrl + '&quot;);">' + builtinEmojiWithColons + '</span>');
+
+        // # Verify that the custom emoji is present in the post
+        cy.getLastPost().find('p').should('contain.html', '<span data-emoticon="' + customEmojiWithoutColons + '">');
+
+        deleteCustomEmoji(testTeam);
+    });
+
+    it('MM-T2187 Custom emoji reaction', () => {
+        const messageText = 'test message';
+
+        // # Open sidebar
+        cy.get('#sidebarHeaderDropdownButton').click();
+
+        // # Click on custom emojis
+        cy.get('#customEmojis').should('be.visible').click();
+
+        // # Click on add new emoji
+        cy.get('.btn-primary').click();
+
+        // # Type emoji name
+        cy.get('#name').type(customEmojiWithColons);
+
+        // # Select emoji image
+        cy.get('input#select-emoji').attachFile(largeEmojiFile);
+
+        cy.get('.backstage-form__footer').within(($form) => {
+            // # Click on Save
+            cy.wrap($form).find('.btn-primary').click();
+        });
+
+        // # Navigate to a channel
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Post a message
+        cy.postMessage(messageText);
+
+        // # Edit post and add a message
+        cy.getLastPostId().then((postId) => {
+            const postText = `#postMessageText_${postId}`;
+            cy.get(postText).should('have.text', messageText);
+
+            // # Edit the last post
+            cy.get('#post_textbox').type('{uparrow}');
+
+            // * Edit post modal should appear, and edit the post
+            cy.get('#editPostModal').should('be.visible');
```

Comment:
Per test case [MM-T2187](https://automation-test-cases.vercel.app/test/MM-T2187), it should be done by post reaction and not by edit modal. Then, please verify if the last post has reaction.
<img width="89" alt="Screen Shot 2020-10-08 at 8 16 52 PM" src="https://user-images.githubusercontent.com/5334504/95457290-39dc7300-09a3-11eb-8e21-9cca2b3878ee.png">